config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  root to: 'keywords#index'
+  
```

Comment:
Lets remove the trailing whitespace

```suggestion

```

I had a comment also in https://github.com/malparty/google-search-ruby/pull/32#issuecomment-858249183

Reply:
@malparty I actually meant to just remove the trailing whitespace as the bot commented [here](https://github.com/malparty/google-search-ruby/pull/32#discussion_r648830728)

It was like the bellow screenshot , we just need to remove the trailing whitespace not the new line as we discussed to keep a new line after the `root route` for better readability 
![Screen Shot 2021-06-10 at 2 30 15 PM](https://user-images.githubusercontent.com/6076762/121483446-66e09a00-c9f8-11eb-85d8-50aab8eb1829.png)

And after removing the trailing whitespace, it will be like 
![Screen Shot 2021-06-10 at 2 23 37 PM](https://user-images.githubusercontent.com/6076762/121483766-c179f600-c9f8-11eb-8cb0-8ad870851c02.png)